pages/js/popup.js

```diff
@@ -16,22 +16,30 @@
  /*global chrome:false */
 
 /**
- * Activate application injection by messaging the background scripting
- * environment. The background scripting environment will then message
- * the privly.js content script.
+ * Helper function for activateExtension() and deactivateExtension().
+ * This makes sure that there is no duplication of code.
```

Comment:
nit: This comment should go in the pull request commentary, but not into the code.